src/nl/hannahsten/texifyidea/lang/commands/LatexDiffcoeffCommand.kt

```diff
@@ -0,0 +1,38 @@
+package nl.hannahsten.texifyidea.lang.commands
+
+import nl.hannahsten.texifyidea.lang.LatexPackage
+import nl.hannahsten.texifyidea.lang.LatexPackage.Companion.DIFFCOEFF
+
+/**
+ * @author Florian Kraft
+ */
+enum class LatexDiffcoeffCommand(
+    override val command: String,
+    override vararg val arguments: Argument = emptyArray(),
+    override val dependency: LatexPackage = LatexPackage.DEFAULT,
```

Comment:
If you put `LatexPackage = DIFFCOEFF` here, you can remove `, dependency = DIFFCOEFF` from the lines below